api_fhir_r4/converters/claimConverter.py

```diff
@@ -27,6 +27,8 @@
 
 from api_fhir_r4.utils import TimeUtils, FhirUtils, DbManagerUtils
 
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
To be able to catch all log in the main logger @edarchis is this the right way ?

```suggestion
logger = logging.getLogger('openimis.' + __name__)
```